lib/form/arWidgetFormUploadQuota.class.php

```diff
@@ -46,6 +50,32 @@ public function render($name, $value = null, $attributes = [], $errors = [])
             .'</label>';
     }
 
+    public function renderB5($name, $value = null, $attributes = [], $errors = [])
+    {
+        $uploadLimit = (int) sfConfig::get('app_upload_limit');
+
+        if (0 === $uploadLimit) {
+            $label = __('%1% upload is disabled', ['%1%' => sfConfig::get('app_ui_label_digitalobject')]);
+        } elseif (-1 === $uploadLimit) {
+            $label = __('Unlimited');
+        } else {
+            $label = __('The uploads directory has not been created yet.');
+
+            $size = Qubit::getDirectorySize(sfConfig::get('sf_upload_dir'));
+            if ($size >= 0) {
+                $label = __(
+                    '%1% used of %2%',
+                    [
+                        '%1%' => hr_filesize($size),
+                        '%2%' => $uploadLimit.' GB',
+                    ]
+                );
+            }
+        }
```

Comment:
It is intentional since we expect the B5 form to take over soon I'm trying to make that process easier for the future developer, i.e. more repetition but less time refactoring hopefully. Do you think that's a good idea?